pkg/reconciler/ingress/ingress.go

```diff
@@ -45,13 +47,22 @@ const (
 	privateLBDomain         = "knative-local-gateway.istio-system.svc.cluster.local"
 	producerServiceName     = "async-producer"
 	asyncOriginalHostHeader = "Async-Original-Host"
+	ingressClassName        = "INGRESS_CLASS_NAME"
+	ingressSuffix           = ".ingress.networking.knative.dev"
 )
 
 // ReconcileKind implements Interface.ReconcileKind.
 func (r *Reconciler) ReconcileKind(ctx context.Context, ing *v1alpha1.Ingress) reconciler.Event {
 	logger := logging.FromContext(ctx)
-	// TODO(bvennam): allow this ingress class to be configurable
-	ingressClass := networkpkg.IstioIngressClassName
+	ingressClass := os.Getenv(ingressClassName)
+
+	if strings.HasSuffix(ingressClass,ingressSuffix) {
+		logger.Debug("valid ingress suffix detected, using ingress class name " + ingressClass)
```

Comment:
Use [`fmt.SPrintf(…)`](https://pkg.go.dev/fmt) here to format the string you pass to the `logger.Debug(…)` call

Reply:
or just `logger.Debugf`